src/Service/Nph/NphOrderService.php

```diff
@@ -663,13 +663,15 @@ public function getRdrObject(NphOrder $order, NphSample $sample, string $type =
         $aliquotsInfo = $this->getAliquots($sample->getSampleCode());
         if ($order->getModule() === '3' && $order->getOrderType() === $order::TYPE_DLW) {
             $dlwInfo = $this->em->getRepository(NphDlw::class)->findOneBy(['module' => $order->getModule(), 'visit' => $order->getVisitType(), 'NphParticipant' => $order->getParticipantId()]);
-            $obj->dlwDose = [
-              'batchid' => $dlwInfo->getDoseBatchId(),
-              'participantweight' => $dlwInfo->getParticipantWeight(),
-              'dose' => $dlwInfo->getActualDose(),
-              'calculateddose' => ($dlwInfo->getParticipantWeight() * 1.5),
-                'doseAdministered' => $dlwInfo->getDoseAdministered()->format('Y-m-d\TH:i:s\Z')
-            ];
+            if (!is_null($dlwInfo) && !empty($dlwInfo)) {
```

Comment:
Could we use a single if condition to check if $dlwInfo is truthy?

```suggestion
            if ($dlwInfo) {
```

Reply:
Sure that works for me.